spec/guard/teaspoon/runner_spec.rb

```diff
@@ -68,6 +61,18 @@
       subject.run([], baz: "teaspoon")
     end
 
+    it 'shows filenames when the show_filenames flag is set' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.